eval/builtin_fn.go

```diff
@@ -780,6 +782,57 @@ func rangeFn(ec *EvalCtx, args []Value, opts map[string]Value) {
 	}
 }
 
+func hasValue(ec *EvalCtx, args []Value, opts map[string]Value) {
+	TakeNoOpt(opts)
```

Comment:
You can declare `var container, key Value` and use `ScanArgs` here. `ScanArgs` also check the number of arguments.